repository/src/lib.rs

```diff
@@ -183,6 +331,9 @@ impl<T: RawRepository> DistributedRepository<T> {
     /// Note that the proof in the `fp` branch must be set for the candidate commit
     /// for the last finalized block, which is `block_commit`.
     pub async fn sync(&mut self, _block_commit: &CommitHash) -> Result<(), Error> {
+        //1. Commithash에 대한 Branch 찾기
```

Comment:
delete TODOs of sync implementation in this PR please

Reply:
fixed.